autopub/create_github_release.py

```diff
@@ -47,5 +48,19 @@ def create_github_release():
         publish=True,
         name=f"{PROJECT_NAME} {version}",
         body=changelog,
-        asset_pattern="dist/*",
+    )
+
+    # give some time to the API to get updated
+    # not sure if this is the proper way to fix the issue
+    # ideally the githubrelease package shouldn't need
+    # to do another API call to get the release since it
+    # should be returned by the create API.
+    # anyway, this fix might be good enough for the time being
+
+    time.sleep(2)
+
+    gh_asset_upload(
+        REPO_SLUG,
+        tag,
+        pattern="dist/*",
     )
```

Comment:
we could use something like tenacity here to make it more resilient, but I expect this code to be remove in future